src/redux/hooks/EditingReservedTokensSplits.ts

```diff
@@ -0,0 +1,31 @@
+import { useAppDispatch } from 'hooks/AppDispatch'
+import { useAppSelector } from 'hooks/AppSelector'
+import { Split } from 'models/splits'
+import { useCallback } from 'react'
+import { editingV2ProjectActions } from 'redux/slices/editingV2Project'
+
+/**
+ * Hook for accessing and setting the redux editing v2 reserved token splits value.
+ */
+export const useEditingReservedTokensSplits = (): [
+  Split[],
+  (input: Split[]) => void,
+] => {
+  const dispatch = useAppDispatch()
+  const { splits } = useAppSelector(
+    state => state.editingV2Project.reservedTokensGroupedSplits,
+  )
+
+  const setSplits = useCallback(
+    (input: Split[]) => {
+      if (!input || !input.length) {
```

Comment:
```suggestion
      if (!input?.length) {
```

perhaps equivalent

Reply:
I think I did this as split[] technically cannot be undefined, but sometimes is (probably a greater bug in our redux code)